File: testprojects/master_scala_main/pipelines/ScalaGCP/code/src/main/scala/graph/OrderBy_2.scala
package graph

import io.prophecy.libs._
import config.ConfigStore._
import udfs.UDFs._
import udfs._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import java.time._

object OrderBy_2 {

  def apply(spark: SparkSession, in: DataFrame): DataFrame =
    in.orderBy(
      col("customer_id").asc,
      col("first_name").asc,
      col("last_name").asc,
      col("phone").asc,
      col("email").asc,
      col("country_code").asc,
      col("account_open_date").asc,
      col("account_flags").asc,
      col("config_values").desc
    )

}

File: testprojects/master_python_mainone/pipelines/PYTHON_SUBGRAPH_CREATOR/code/job/graph/MultiPort/OrderBy_2.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from prophecy.libs import typed_lit
from job.config.ConfigStore import *
from job.udfs.UDFs import *

def OrderBy_2(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0.orderBy(
        col("`- c long`").asc(), 
        col("`c- short`").asc(), 
        col("`c -  boolean _  `").asc(), 
        col("c_double").asc()
    )
